manifests/params.pp

```diff
@@ -22,6 +22,7 @@
   $nx_confd_purge        = false
   $nx_worker_processes   = 1
   $nx_worker_connections = 1024
+  $nx_ignore_invalid_headers = on
```

Comment:
Sure, I can hieraize it in modules/bioiq_common/manifests/sso/lb.pp where Lyle passed in ignore_invalid_headers => 'off' in https://github.com/BioIQ/puppet/pull/950/files.
params.pp is part of the standard nginx module so we shouldn't hieraize it here, nx_ignore_invalid_headers = on is just the default like the rest of the params.